BizHawk.Client.EmuHawk/ArgParser.cs

```diff
@@ -28,6 +28,9 @@ class ArgParser
 		public int socket_port = 9999;
 		public string socket_ip = null;
 		public string run_id = null;
+		public bool use_two_controllers = false;
+		public int socket_port_p2 = 10000;
+		public string socket_ip_p2 = "127.0.0.1";
```

Comment:
why isn't this null?

Reply:
because we should have sane default for ports/etc for the 2nd player, so you can just pass in use_two_controllers=true